CONTRIBUTING.md

```diff
@@ -57,8 +57,14 @@ You can add the sign off when creating the git commit via `git commit -s`.
 
 ## Code Reviews
 
-* A single approval is sufficient to merge. If a reviewer asks for changes in a PR they should be
-addressed before the PR is merged, even if another reviewer has already approved the PR.
-* During the review, address the comments and commit the changes _without_ squashing the commits.
-This facilitates incremental reviews since the reviewer does not go through all the code again to
-find out what has changed since the last review.
+* A single approval is sufficient to merge. If a reviewer asks for
+changes in a PR they should be addressed before the PR is merged,
+even if another reviewer has already approved the PR.
+* During the review, address the comments and commit the changes
+_without_ squashing the commits. This facilitates incremental reviews
+since the reviewer does not go through all the code again to find out
+what has changed since the last review.
```

Comment:
```suggestion
what has changed since the last review. When a change goes out of sync with main,
please rebase and force push, keeping the original commits where practical.
```